client/views/admin/customEmoji/AddCustomEmoji.js

```diff
@@ -30,14 +32,23 @@ function AddCustomEmoji({ close, onChange, ...props }) {
 
 	const handleSave = useCallback(async () => {
 		const formData = new FormData();
+
+		const regExp = new RegExp('^[a-zA-Z0-9_]+$');
 		formData.append('emoji', emojiFile);
-		formData.append('name', name);
-		formData.append('aliases', aliases);
-		const result = await saveAction(formData);
 
-		if (result.success) {
-			onChange();
-			close();
+		if (regExp.test(name) && regExp.test(aliases)) {
+			formData.append('name', name);
+			formData.append('aliases', aliases);
+			const result = await saveAction(formData);
+			if (result.success) {
+				onChange();
+				close();
+			}
+		} else {
+			dispatchToastMessage({
+				type: 'error',
+				message: t('You_Can_not_use_special_characters_in_custom_emoji'),
```

Comment:
Change the error message to something simpler, like `custom_emoji_special_characters` and make sure to add at least english translations for this :)

Reply:
@KevLehman please review the changes